app/src/main/java/org/eyeseetea/malariacare/layout/listeners/LogoutAndLoginRequiredOnPreferenceClickListener.java

```diff
@@ -32,7 +31,7 @@ public boolean onPreferenceClick(final Preference preference) {
         return activity.onPreferenceClick(preference);
     }
 
-    public static void logout(final Activity activity) {
+    public void logout() {
```

Comment:
@idelcano this class is created from SettingsActivityStrategy in lao and cambodia but is not used, has a logout method but never is invoked